floor/lib/src/adapter/insertion_adapter.dart

```diff
@@ -36,7 +36,7 @@ class InsertionAdapter<T> {
     final OnConflictStrategy onConflictStrategy,
   ) async {
     if (items.isEmpty) return;
-    await _insertList(items, onConflictStrategy);
+    await _insertListVoid(items, onConflictStrategy);
```

Comment:
💡 You could inline the functionality as it's only used once. You also need to make sure to map `OnConflictStrategy` to sqflite's `ConflictAlgorithm`.

```suggestion
    final batch = _database.batch();
    for (final item in items) {
      batch.insert(
        _entityName,
        _valueMapper(item),
        conflictAlgorithm: onConflictStrategy.asSqfliteConflictAlgorithm(),
      );
    }

    await batch.commit(noResult: true);

    if (_changeListener != null) {
      _changeListener.add(_entityName);
    }
  }
```